src/win/util.c

```diff
@@ -1676,14 +1676,27 @@ int uv_os_gethostname(char* buffer, size_t* size) {
     return uv_translate_sys_error(WSAGetLastError());
 
   buf[sizeof(buf) - 1] = '\0'; /* Null terminate, just to be safe. */
-  len = strlen(buf);
+  int utf16StrBufSize = MultiByteToWideChar(CP_ACP, 0, buf, -1, NULL, 0);
```

Comment:
Style: snake_case, not camelCase (i.e., `utf16_str_buf_size`), and declare variables at the top of the function.

Reply:
Done